Add content-based message routing tutorial (#7862)
ballerina-platform/ballerina-dev-website: swan-lake/integration-tutorials/content-based-message-routing.md

---
layout: ballerina-content-based-message-routing-left-nav-pages-swanlake
title: Content-based message routing 
permalink: /learn/content-based-message-routing/
description: Integration tutorial for message routing based on the content.
keywords: ballerina, programming language, client, restful-api, integration
active: content-based-message-routing

> intro: This tutorial helps you understand the basics of how ballerina can be used to write a single service which route the requests based on the content.

Reviewer: ```suggestion
intro: This tutorial helps you understand how content-based routing can be done in Ballerina.
```
